packages/orm/src/decorators/manyToMany.ts

```diff
@@ -0,0 +1,28 @@
+import { many } from "..";
+import { AnyModel } from "../Model";
+import { ModelClassTypeFromModelFields, ModelName, PossibleFieldKeys } from "../types";
+import { registerDescriptor } from "./utils";
+
+/**
+ * A decorator that registers a many-to-many relationship between
+ * this (source) and another (target) model.
+ *
+ * The first argument is the name
+ * of the Model the key is pointing to, and
+ * the second one is an optional related name, which will
+ * be used to access the Model the key
+ * is being defined from, from the target Model.
+ *
+ * If the related name is not passed, it will be set as
+ * `${toModelName}Set`.
+ * 
+ * @param toModelName The model name you want to define a relation to.
+ * @param relatedName The name of key, you want to create backwards relation for.
+ */
+export function ManyToMany<MClass extends AnyModel>(
```

Comment:
I did not extend the exposed API here that would accept the `opts` as an argument. We have a few usages of this feature in partners-spa and all can be changed to the one passing two strings instead.

Reply:
Same for other relational fields decorators.

Reply:
However, I can extend this as I did in `registerDescriptor` function.